src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -0,0 +1,111 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+    @Override
+    public Driver create(Driver driver) {
+        String insertDriverRequest = "INSERT INTO drivers(name, license_number) VALUES(?, ?);";
```

Comment:
For SQL it's better to use "query" instead of "request"
```suggestion
        String insertDriverQuery = "INSERT INTO drivers(name, license_number) VALUES(?, ?);";
```